Source/OxyPlot/Axes/Axis.cs

```diff
@@ -1679,31 +1697,6 @@ protected double CalculateActualInterval(double availableSize, double maxInterva
         }
 
         /// <summary>
-        /// Calculates the minor interval.
-        /// </summary>
-        /// <param name="majorInterval">The major interval.</param>
-        /// <returns>The minor interval.</returns>
-        protected double CalculateMinorInterval(double majorInterval)
-        {
-            // if major interval is 100, the minor interval will be 20.
-            return majorInterval / 5;
-
-            // The following obsolete code divided major intervals into 4 minor intervals, unless the major interval's mantissa was 5.
-            // e.g. Major interval 100 => minor interval 25.
-
-            // Func<double, double> exponent = x => Math.Ceiling(Math.Log(x, 10));
-            // Func<double, double> mantissa = x => x / Math.Pow(10, exponent(x) - 1);
-            // var m = (int)mantissa(majorInterval);
-            // switch (m)
-            // {
-            // case 5:
```

Comment:
Yes, I guess it would do the exact same thing. I find my code to be more intuitive, but then again, I wrote it, so I'm not to judge. I guess you choose which one you like better (and I'm not mad either case) :)